### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by artberger**
# Description

Please include a summary of the changes.

This bug fixes ... \ This new feature can be used to ...

# Context

Updates docs for various items, such as bumping the version, reviewing a dev guide, and adding more information about version support.

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- N/A If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- N/A I opened a draft PR or added the work in progress label if my PR is not ready for review
- N/A I have performed a self-review of my own code
- N/A I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- N/A I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/content/guides/dev/building-and-deploying-gloo.md [hunk 1]
```diff
@@ -3,131 +3,185 @@ title: "Building and Deploying Gloo Edge from Source"
...
 ---
 
-## Building Gloo
+You can build and deploy Gloo Edge Open Source from the source code.
 
-### Setup
+## Before you begin
 
-Before going through this guide, readers should go through the [setup guide](https://docs.solo.io/gloo-edge/latest/guides/dev/setting-up-dev-environment/) and ensure the dependencies in this section are installed. After this, all products in Gloo can be built from the top-level [Makefile](https://github.com/solo-io/gloo/blob/master/Makefile).
+1. Follow the [setup guide](https://docs.solo.io/gloo-edge/latest/guides/dev/setting-up-dev-environment/) to clone the Gloo repository and install the project dependencies.
```
Comment (Rachael-Graham) [inline docs/content/guides/dev/building-and-deploying-gloo.md:10]:
```suggestion
1. Follow the [setup guide]({{% versioned_link_path fromRoot="/guides/dev/setting-up-dev-environment/" %}}) to clone the Gloo repository and install the project dependencies.
```

File: docs/content/guides/dev/building-and-deploying-gloo.md [hunk 2]
```diff
@@ -3,131 +3,185 @@ title: "Building and Deploying Gloo Edge from Source"
...
+2. In your terminal, navigate to the Gloo project.
+   ```sh
+   cd ${GOPATH}/src/github.com/solo-io/gloo
+   ```
+3. Continue with this guide to learn how you can build all Gloo from the top-level [Makefile](https://github.com/solo-io/gloo/blob/master/Makefile).
```
Comment (Rachael-Graham) [inline docs/content/guides/dev/building-and-deploying-gloo.md:15]:
```suggestion
3. Continue with this guide to learn how you can build Gloo Edge from the top-level [Makefile](https://github.com/solo-io/gloo/blob/master/Makefile).
```

File: docs/content/guides/dev/building-and-deploying-gloo.md [hunk 3]
```diff
@@ -3,131 +3,185 @@ title: "Building and Deploying Gloo Edge from Source"
...
 
-### Build Setup and Code Generation
+## Set up build dependencies and code generation {#setup}
 
-Start by running
+From the Gloo project directory, run:
```
Comment (Rachael-Graham) [inline docs/content/guides/dev/building-and-deploying-gloo.md:19]: add code ticks around the following command

File: docs/content/guides/dev/building-and-deploying-gloo.md [hunk 4]
```diff
@@ -3,131 +3,185 @@ title: "Building and Deploying Gloo Edge from Source"
...
     make -B install-go-tools generated-code
 
-The `install-go-tools` target will install dependencies necessary to build gloo in `_output/.bin`. The `generated-code` target does multiple things:
+The `install-go-tools` target installs the dependencies to build Gloo in `_output/.bin`. 
 
-1. It generates the go source code from the `.proto` files in the `api` directories. For example, `.proto` files in `projects/gloo/api` will be generated in `projects/gloo/pkg/api`
+The `generated-code` target does multiple things:
 
-2. It generates the CRDs used to interact with Gloo via kubernetes. These CRDs can be found in `install/helm/gloo/crds`.
+1. It generates the Go source code from the `.proto` files in the `api` directories. For example, `.proto` files in `projects/gloo/api` will be generated in `projects/gloo/pkg/api`
```
Comment (Rachael-Graham) [inline docs/content/guides/dev/building-and-deploying-gloo.md:27]:
```suggestion
1. It generates the Go source code from the `.proto` files in the `api` directories. For example, `.proto` files in `projects/gloo/api` will be generated in `projects/gloo/pkg/api`.
```

File: docs/content/guides/dev/building-and-deploying-gloo.md [hunk 5]
```diff
@@ -3,131 +3,185 @@ title: "Building and Deploying Gloo Edge from Source"
...
+| `gloo-docker` | The name for the Docker image. |
 
-Once the unit tests pass, the next step is to rebuild the docker image and update the cluster as described above. Any output printed to stdout can be viewed by running
+Example output:
 
-    kubectl -n gloo-system logs deployments/gloo
+    Successfully tagged quay.io/solo-io/gloo:0.0.1
 
-Since this is fairly laborious, it is a good idea to make sure that the tests are updated and passing before going to this step.
+## Deploy Gloo {#deploy}
```
Comment (Rachael-Graham) [inline docs/content/guides/dev/building-and-deploying-gloo.md:59]: I think we need to specify gllo edge to not confuse w gloo platform where possible
```suggestion
## Deploy Gloo Edge {#deploy}
```

File: docs/content/guides/dev/building-and-deploying-gloo.md [hunk 6]
```diff
@@ -3,131 +3,185 @@ title: "Building and Deploying Gloo Edge from Source"
...
+
+You can choose from [several Gloo installation options]({{% versioned_link_path fromRoot="/installation/preparation/#deployment-requirements" %}}). This guide assumes you deploy Gloo into a Kubernetes cluster that runs locally in [kind]({{% versioned_link_path fromRoot="/installation/platform_configuration/cluster_setup/#kind" %}}).
```
Comment (Rachael-Graham) [inline docs/content/guides/dev/building-and-deploying-gloo.md:61]:
```suggestion
You can choose from [several Gloo Edge installation options]({{% versioned_link_path fromRoot="/installation/preparation/#deployment-requirements" %}}). This guide assumes you deploy Gloo Edge into a Kubernetes cluster that runs locally in [kind]({{% versioned_link_path fromRoot="/installation/platform_configuration/cluster_setup/#kind" %}}).
```

**Review by Rachael-Graham — APPROVED** [review]

**Review by Nadine2016 — APPROVED** [review]

**Pull request merged**